common/test/acceptance/pages/studio/utils.py

```diff
@@ -304,5 +306,5 @@ def get_side_bar_help_element_and_click_help(self, as_list_item=False):
         else:
             element_css = SIDE_BAR_HELP_CSS
 
-        self.q(css=element_css).click()
-        return self.q(css=element_css).results[0]
+        self.q(css=element_css).results[index].click()
+        return self.q(css=element_css).results[index]
```

Comment:
@Muddasser what exactly we are doing here? Instead of executing the query twice you can do the below

```Python
element = self.q(css=element_css).results[index]
element.click()
return element
```